src/SubtitleOctopus.cpp

```diff
@@ -124,12 +247,34 @@ class SubtitleOctopus {
         track = NULL;
         canvas_w = 0;
         canvas_h = 0;
+        drop_animations = false;
+        scanned_events = 0;
     }
 
     void setLogLevel(int level) {
         log_level = level;
     }
 
+    void setDropAnimations(int value) {
+        drop_animations = !!value;
+        if (drop_animations)
+            scanAnimations(scanned_events);
+    }
+
+    /*
+     * \brief Scan events starting at index i for animations
+     * and discard animated tags when found.
+     * Note that once animated tags were dropped they cannot be restored.
+     * Updates the class memeber scanned_events to last scanned index.
+     */
+    void scanAnimations(int i) {
+        ASS_Event *cur = track->events;
```

Comment:
unused?

Reply:
Appears to have been a leftover from a previous version. Now removed, thanks.